src/Telegram.Bot/Types/ReactionType.cs

```diff
@@ -0,0 +1,62 @@
+using Telegram.Bot.Converters;
+
+namespace Telegram.Bot.Types;
+
+/// <summary>
+/// This object describes the type of a reaction. Currently, it can be one of
+/// <list type="bullet">
+/// <item><description>ReactionTypeEmoji</description></item>
+/// <item><description>ReactionTypeCustomEmoji</description></item>
+/// </list>
+/// </summary>
+[JsonObject(MemberSerialization.OptIn, NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
+[JsonConverter(typeof(ReactionTypeConverter))]
+public abstract class ReactionType
+{
+    /// <summary>
+    /// Type of the reaction
+    /// </summary>
+    [JsonProperty]
+    public abstract string Type { get; }
+}
+
+/// <summary>
+/// The reaction is based on an emoji.
+/// </summary>
+[JsonObject(MemberSerialization.OptIn, NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
+public class ReactionTypeEmoji : ReactionType
+{
+    /// <summary>
+    /// Type of the reaction, always "emoji"
+    /// </summary>
+    public override string Type => "emoji";
+
+    /// <summary>
+    /// Reaction emoji. Currently, it can be one of "👍", "👎", "❤", "🔥", "🥰", "👏", "😁",
```

Comment:
I think we should add a static class with const fields for each emoji to make it more discoverable. You can look names for each emoji here http://emojipedia.org
```csharp
public static class KnownReactionTypeEmoji
{
    public const string ThumbsUp = "👍";
    // etc
}
```

Reply:
I'll die on that hill :(

Reply:
addressed